modin/pandas/base.py

```diff
@@ -2322,7 +2322,15 @@ def rmod(
             "rmod", other, axis=axis, level=level, fill_value=fill_value
         )
 
-    rmul = mul
+    def rmul(
+        self, other, axis="columns", level=None, fill_value=None
+    ):  # noqa: PR01, RT01, D200
+        """
+        Get Multiplication of dataframe and other, element-wise (binary operator `rmul`).
```

Comment:
```suggestion
        Get Multiplication of `other` and `self` dataframe, element-wise (binary operator `rmul`).
```
swap the operands in docstring to highlight that it's reversed mul?

Reply:
I agree that what you suggest is clearer, but I would like to keep this as is to match the pandas API documentation.